runtime/parachains/src/inclusion/tests.rs

```diff
@@ -1555,7 +1555,7 @@ fn backing_works() {
 		let backed_c = block_on(back_candidate(
 			candidate_c.clone(),
 			&validators,
-			group_validators(GroupIndex::from(2)).unwrap().as_ref(),
+			group_validators(GroupIndex::from(2 as usize)).unwrap().as_ref(),
```

Comment:
Why not 2_usize? And why not have a `From` instance for i32?

Reply:
Not sure but I think implementing a From instance for i32 would still lead to the same issue. The compiler was having issues deciding which number type to use since u32 and usize were both available.

How do I implement 2_usize (to usize?) (sorry about the beginner question)